etc/kayobe/ansible/requirements.yml

```diff
@@ -14,18 +12,19 @@ collections:
     version: 1.1.0
 roles:
   - src: stackhpc.vxlan
-  - name: ansible-lockdown.ubuntu22_cis
-    src: https://github.com/ansible-lockdown/UBUNTU22-CIS
-    version: 1.4.1
+  - name: ansible-lockdown.ubuntu24_cis
+    src: https://github.com/ansible-lockdown/UBUNTU24-CIS
+    version: 1.0.0
   - name: ansible-lockdown.rhel9_cis
     src: https://github.com/ansible-lockdown/RHEL9-CIS
-    version: 1.3.1
+    version: v1.3.4
   - name: wazuh-ansible
     src: https://github.com/stackhpc/wazuh-ansible
     version: stackhpc-v4.10.0
   - name: geerlingguy.pip
-    version: 2.2.0
+    version: 3.1.0
   - name: monolithprojects.github_actions_runner
-    version: 1.18.5
+    version: 1.21.1
```

Comment:
This doesn't match version 1.25.1 mentioned in the release note.

Reply:
Fixed